_data/simple-icons.json

```
@@ -12975,6 +12975,11 @@
             "hex": "000000",
             "source": "https://github.com/swc-project/logo/blob/f26cac1b4a490e3bdf128d3b084bb57f4fab1aac/svg/swc_black.svg"
         },
+        {
+            "title": "SWI Prolog",
+            "hex": "ED1C24",
+            "source": "https://seekicon.com/free-icon/prolog_1"
```

```suggestion
            "source": "https://www.swi-prolog.org/icons/vector/swipl-logo.svg"
```
We would prefer to use official source always when available.